### ISSUE TITLE
Support opening request/response content in system application

### ISSUE THREAD

**Issue by mfulton26** [THemming/harviewer-chrome-app#14]
It would be great to have a link/button for Inspect entries to open the request/response content in the operating system application registered for that type. e.g. XML, JSON, TXT, PNG, etc. files might all be registered on the operating system to open in an IDE so clicking the link/button would extract the content from the HAR file, create a new/temp file, and open it in the IDE. This reduces the need to support viewing many different types in the HTTP Archive Viewer itself while giving users an easy way to open/view content.

**Comment by THemming** [comment]
Good idea, thanks. I'll look into the feasibility of this and pencil it in for the 1.5 release.
